jupyterlab_pullrequests/managers/github.py

```diff
@@ -204,7 +201,7 @@ async def list_prs(self, username: str, pr_filter: str) -> List[Dict[str, str]]:
             )
 
         # Reset cache
-        self._pull_requests_cache = {}
+        self._pull_requests_cache = None
```

Comment:
Why do you reset the cache to `None` and not an empty dictionary? The idea to set as an empty dictionary is to allow to use the get method line 298.